File: 프로그래머스/1/12977. 소수 만들기/소수 만들기.js
function isPrime(a, b, c){
    const n = a + b + c;
    for(let i = 2 ; i <= Math.floor(n / 2) ; i++){
        if(n % i === 0) return false;
    }
    return true;
}

function solution(nums) {
    let result = 0;
    for(let i = 0 ; i < nums.length - 2 ; i++){
        for(let j = i+1 ; j < nums.length - 1 ; j++){
            for(let k = j+1 ; k < nums.length ; k++){
                if(isPrime(nums[i], nums[j], nums[k])) result++;
            }
        }
    }
    return result;
}

File: 프로그래머스/1/12977. 소수 만들기/소수 만들기.py
from itertools import combinations
def isPrime(a, b, c):
    n = a + b + c
    for i in range(2, n // 2 + 1):
        if n % i == 0:
            return False
    return True

def solution(nums):
    combi_nums = combinations(nums, 3)
    cnt = 0
    for a, b, c in combi_nums:
        if isPrime(a, b, c):
            cnt += 1
            
    return cnt
    